komorebi.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import gi
+import signal
+import sys
+import logging
+
+gi.require_versions({
+    'Gtk': '3.0',
+    'GObject': '2.0',
+    'GtkClutter': '1.0',
+    'Clutter': '1.0',
+    'ClutterGst': '3.0',
+    'Gdk': '3.0',
+    'GdkPixbuf': '2.0',
+    'Gst': '1.0',
+    'WebKit2': '4.0',
+})
+
+from gi.repository import Gio, Gtk, GtkClutter, Gdk, Gst, Clutter
+
+from komorebi.preferences_window import PreferencesWindow
+from komorebi.settings import ConfigKeys, Settings
+import komorebi.utilities
+
+from komorebi.screen import Screen
+
+
+def check_desktop_compatible():
+    return not('wayland' in os.environ.get('XDG_SESSION_TYPE', '') or os.environ.get('WAYLAND_DISPLAY', ''))
```

Comment:
This seems weird, are you sure you don't want to do `os.environ.get('XDG_SESSION_TYPE') == 'wayland'`?

```python
>>> 'wayland' in 'lalalawaylandlalala'
True
```

Also, the default argument fo `os.environ.get` can be omitted, because it will fallback to `None`.

```python
>>> os.environ.get('does-not-exist')
None
```

Reply:
Yes, this was one of the first files I started with, so my Python knowledge was still very bad :stuck_out_tongue: